interactable_object.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "type": 222,
+        "permit": "allow",
+        "location": {
+            "x": 459,
+            "y": 170,
+            "z": 663
+          }
+    },
+    {
+        "type": 161,
+        "permit": "deny",
+        "location": {
+            "x": 453,
+            "y": 167,
+            "z": 665
+          },
+        "custom_message": "``This door seem closed, you maybe need a key``"
+    },
+    {
+        "type": 222,
+        "location": "*",
+        "permit": "deny"
+    }
```

Comment:
What is the purpose of this?

Reply:
It's to cancel every trapdoor because the majority of trapdoor is for decoration so i deny all and u just need to add specific trapdoor rule just above

Reply:
Ok, I think you can handle this directly in the code. The json should contain only interactive objects